content/en/tracing/profiler/search_profiles.md

```diff
@@ -184,6 +200,22 @@ Once enabled, the following profile type is collected:
 Wall Time
 : Shows the elapsed time spent in managed methods. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the method is running. 
 
+{{< /programming-lang >}}
+{{< programming-lang lang="php" >}}
+
+Once enabled, the following profile types are collected:
+
+Wall Time
+: Shows the elapsed time used by each function. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the function is running.
+
```

Comment:
We have an env var `DD_PROFILING_EXPERIMENTAL_CPU_ENABLED` which controls whether CPU Time is gathered. It's biased towards I/O functions but it's cheap. Not sure if it's worth mentioning something here, or if we should wait.

Reply:
Let's wait till it's on by default.